test/time-math.js

```diff
@@ -0,0 +1,90 @@
+const test = require('tape');
+const {
+    minutesToHourFormated,
+    subtractTime,
+    sumTime,
+    validateTime
+} = require('../js/time-math.js');
+
+// Format hours test
+test('Minues to Hours Formatting', (t) => {
+    t.assert(minutesToHourFormated(1) === '00:01', '1 ==> '
+        + minutesToHourFormated(1));
+    t.assert(minutesToHourFormated(0) === '00:00', '0 ==> '
+        + minutesToHourFormated(0));
+    t.assert(minutesToHourFormated(-1) === '-00:01', '-1 ==> '
+        + minutesToHourFormated(-1));
+    t.assert(minutesToHourFormated(-59) === '-00:59', '-59 ==> '
+        + minutesToHourFormated(-59));
+    t.assert(minutesToHourFormated(-60) === '-01:00', '-60 ==> '
+        + minutesToHourFormated(-60));
+    t.assert(minutesToHourFormated(-61) === '-01:01', '-61 ==> '
+        + minutesToHourFormated(-61));
+    t.assert(minutesToHourFormated(59) === '00:59', '59 ==> '
+        + minutesToHourFormated(50));
+    t.assert(minutesToHourFormated(60) === '01:00', '60 ==> '
+        + minutesToHourFormated(60));
+    t.assert(minutesToHourFormated(61) === '01:01', '61 ==> '
+        + minutesToHourFormated(61));
+    t.end();
+});
+
+// Subtract time
+test('Subtract Time', (t) => {
+    t.assert(subtractTime('1:00', '1:00') === '00:00', '1:00 - 1:00 = '
+        + subtractTime('1:00', '1:00'));
+    t.assert(subtractTime('00:00', '00:00') === '00:00', '00:00 - 00:00 = '
+        + subtractTime('00:00', '00:00'));
+    t.assert(subtractTime('00:01', '01:00') === '00:59', '01:00 - 00:01 = '
+        + subtractTime('00:01', '01:00'));
+    t.assert(subtractTime('13:00', '12:00') === '-01:00', '12:00 - 13:00 = '
+        + subtractTime('13:00', '12:00'));
+    t.assert(subtractTime('48:00', '24:00') === '-24:00', '24:00 - 48:00 = '
+        + subtractTime('48:00', '24:00'));
+    t.assert(subtractTime('00:01', '12:00') === '11:59', '12:00 - 00:01 = '
+        + subtractTime('00:01', '12:00'));
+    t.assert(subtractTime('13:00', '12:00') === '-01:00', '13:00 - 12:00 = '
```

Comment:
Should be:
```suggestion
    t.assert(subtractTime('13:00', '12:00') === '-01:00', '12:00 - 13:00 = '
```

In any case, it is exactly like line 40

Reply:
This one is correct, the function subtracts the first argument from the second.

Reply:
Yes, that's what I meant, it does 12-13, but the message is sayng 13-12.

But it is a test already covered by line 40

Reply:
I solved this by reversing the second one, no more duplicate.